spec.md

```diff
@@ -193,6 +214,53 @@ To “close” a node $N_i$:
 
 # Rolling Hash Functions
 
+## CP32
+
+The `cp32` hash function is based on cyclic polynomials. The family of
+related functions is sometimes also called "buzhash." `cp32` is the
+recommended hash function for use with hashsplit; use it unless you have
+clear reasons for doing otherwise.
```

Comment:
It would be good to add some rationale here. (Perhaps in a subsequent PR.) Why is cp32 recommended? Why not rrs, which from the description below sounds much more common and therefore a likelier standard?

(These are rhetorical questions - I know the answer.)